lineman/app/components/thread_preview/thread_preview.haml

```diff
@@ -6,10 +6,13 @@
     %button.btn.btn-default.btn-sm.thread-preview__mark-as-read{ng-click: 'thread.markAsRead()', ng-class: '{disabled: !thread.isUnread()}'}>
       %i.fa.fa-check>
       %span{translate: 'dashboard_page.mark_as_read'}
-    %button.btn.btn-default.btn-sm.thread-preview__mute{ng-click: 'thread.changeVolume("mute")', ng-class: '{disabled: thread.volume == "mute"}'}>
+    %button.btn.btn-default.btn-sm.thread-preview__mute{ng-click: 'changeVolume("mute")', ng-show: 'thread.volume != "mute"'}>
       %i.fa.fa-volume-off
       %i.fa.fa-times>
       %span{translate: 'volume_levels.mute'}
+    %button.btn.btn-default.btn-sm.thread-preview__mute{ng-click: 'changeVolume("normal")', ng-show: 'thread.volume == "mute"'}>
```

Comment:
How does the suggestion to abstract the comparision into a function land with you? IE: thread.isMuted()